components/prism-php.js

```diff
@@ -114,6 +126,25 @@
 	// The different types of PHP strings "replace" the C-like standard string
 	delete Prism.languages.php['string'];
 
+	Prism.languages.insertBefore('php', 'class-name', {
+		'class-static-call': {
+			pattern: /[a-zA-Z_][\w\\]+(?=\:\:)/,
```

Comment:
1. We want to support single-character class names.
1. There may be any number of spaces before `::`.
1. The `\b` will prevent false positives.

```suggestion
			pattern: /\b[a-zA-Z_][\w\\]*(?=\s*::)/,
```

Reply:
Same as namespace, do not  match class names beginning with number -> fixed in next commit